projects/plugins/wpcomsh/wpcom-themes/includes/class-wpcom-themes-mapper.php

```diff
@@ -71,6 +71,7 @@ public function map_wpcom_to_wporg( stdClass $wpcom_theme ): stdClass {
 		$theme->creation_time        = $wpcom_theme->date_added;
 		$theme->is_wpcom_theme       = true;
 		$theme->tags                 = $this->build_theme_tags( $wpcom_theme );
+		$theme->theme_tier           = $wpcom_theme->theme_tier->slug ?? null;
```

Comment:
Maybe we can default to `Community theme` or something more meaningful than `null`?